### DISCUSSION TITLE
翻訳部分のデザイン

### DISCUSSION THREAD

**Discussion by ttizze** [ttizze/eveeve#68] [Ideas]
![image](https://github.com/user-attachments/assets/07feab25-aa3b-4919-b461-ba68850577a2)

この部分のデザイン
eveeveは原文と訳文を読み比べやすくしたい｡
現在のデザインだと訳文が浮いている｡

**Comment by linear** [comment]
<p><a href="https://linear.app/eveeve/issue/STE-103/翻訳部分のデザイン">STE-103 翻訳部分のデザイン</a></p>

**Comment by ttizze** [comment]
![image](https://github.com/user-attachments/assets/8b2d9987-eb5f-4e93-9873-b5c9e6b8a2e8)

**Comment by ttizze** [comment]
![image](https://github.com/user-attachments/assets/f2f2bcc7-f216-440e-bcec-6fdec677c28f)

**Comment by ttizze** [comment]
![image](https://github.com/user-attachments/assets/920add62-652d-4ad3-b9a5-31753ea615b1)

**Comment by ttizze** [comment]
![image](https://github.com/user-attachments/assets/e62184f6-687d-4ec5-b614-f392830a9207)

**Comment by ttizze** [comment]
![image](https://github.com/user-attachments/assets/24a838dc-95c0-410b-906e-2632afe99ceb)

**Comment by ttizze** [comment]
![image](https://github.com/user-attachments/assets/0ca67f3d-1040-4737-87fa-e109e69d384b)

**Comment by ttizze** [comment]
上にスクロールした時だけ原文のみ表示とか切り替えるメニューを出す
原文を薄くする
タップで翻訳メニュー出す
